app/containers/Avatar.js

```diff
@@ -23,7 +24,14 @@ const Avatar = React.memo(({
 		type, text, size, userId, token, avatar, baseUrl
 	});
 
-	let image = (
+	let image = emoji ? (
+		<Markdown
+			msg={emoji}
+			baseUrl={baseUrl}
+			getCustomEmoji={getCustomEmoji}
+			theme={theme}
+		/>
```

Comment:
This could be a FastImage directly

Reply:
Actually, `CustomEmoji` is giving blank output for `emojiUnicode`. It is giving output for custom emojies